### PR TITLE
[py-gradio] New package

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-gradio/package.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    depends_on("py-jinja2", type=("build", "run"))
+    depends_on("py-markdown-it-py@2.0.0:+linkify", type=("build", "run"))
+    depends_on("py-pygments@2.12.0:", type=("build", "run"))
+    depends_on("py-mdit-py-plugins@:0.3.3", type=("build", "run"))
+    depends_on("py-markupsafe", type=("build", "run"))
+    depends_on("py-matplotlib", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-orjson", type=("build", "run"))
+    depends_on("py-pandas", type=("build", "run"))
+    depends_on("py-pillow", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-gradio/package.py]:
```suggestion
    depends_on("pil", type=("build", "run"))
```
We have a virtual dep for this

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by qwertos** [comment]
Accidentally pushed out of order. Drafting until dependencies get accepted.

**Comment by adamjstewart** [comment]
You can always include multiple packages in a single PR if they are co-dependent

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Let me know when this is "Ready for review" and I'll merge.

**Comment by qwertos** [comment]
All set

**Pull request merged**